packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/java/redirecturi.md

```diff
@@ -1,3 +1,3 @@
-If you are setting up the sample app, use this value:
-`http://localhost:8080`. Otherwise, choose
-a URI that is used by your app.
+If you are setting up the embedded authentiation sample app, use `http://localhost:8080` as the redirect URI. Otherwise, use a redirect URI that is appropriate for your app.
+
+> **Note:** The default redirect URI, `http://localhost:8080/authorization-code/callback` is listed when you create a new app integration in the Admin Console. Do not remove this redirect URI.
```

Comment:
Did you verify this default redirect URI?

Reply:
I added the default redirect URI (in addition to the `http://localhost:8080`) in the Admin Console, and ran the app with redirectURI=`http://localhost:8080`, and everything still works as before for the widget sample app.

Reply:
```suggestion
```

Reply:
looks good.